content/02.introduction.md

```diff
@@ -24,69 +24,58 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
-Other aspects of transmission to investigate are the relationship between infectiousness and virus shedding with disease period or symptoms and also the proportions of cases that are attributable to various types of transmission events, such as transmission between relatives, nosocomial transmissions, and other possible types of interactions.
-Some information about these characteristics of transmission are available for the highly pathogenic coronaviruses SARS-CoV and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
-For SARS-CoV-2 it is still being investigated whether, in addition to being spread by people who show symptoms,  the virus can be transmitted by people who do not show symptoms [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html], such as during the pre-symptomatic stage of infection or in people with asymptomatic infections (a presentation which sometimes occurs in other infectious diseases but is still being investigated for SARS-CoV-2).
-
-### SARS-CoV-2: Origin, Phylogenetics, and Evolution
-
-#### Origin of SARS-CoV-2
-The evolutionary origins of the SARS-CoV-2 virus are not yet fully understood.
-Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
-
-Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
-This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
-
-### COVID-19: Mechanisms and Presentation
-
-#### Immunological Mechanisms of Coronavirus-driven Disease in Humans
-
-Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+### COVID-19: Infection and Presentation
+
+#### Pathogenic Mechanisms of Coronaviruses
+
+Until more information becomes available about the pathogenesis of SARS-CoV-2 specifically, insight can be gained by examining how the human body responds to viral pathogens generally, how infection from related viruses proceeds, and how this information intersects with what is known about COVID-19. 
+The human immune response is double-pronged, with the innate and adaptive immune responses playing distinct roles.
```

Comment:
Since you describe the types of adaptive immunity, it may be worth also describing briefly here the types of innate immunity. Suggested sentence:
Innate immunity includes barriers (such as skin and other epithelial surfaces, and mucus), innate defenses by most cells (such as through membrane-bound pattern-recognition receptors, and the interferon response), the soluble proteins of the complement system which have roles in innate defense, and immune cells which have roles in innate defense (such as macrophages and natural killer cells).